slitlessutils/examples/wr96.py

```diff
@@ -46,6 +47,21 @@
 SUFFIX, DRZSUF = 'flc', 'drc'
 SCALE = 0.05          # driz image pix scale
 
+WR96_DRIZZLE_PARAMS = {
```

Comment:
Done in https://github.com/spacetelescope/slitlessutils/pull/27/commits/7b102cf9f395e3e0c38ce42c2fe4f40ebf319fdf. I also hid the dictionary args so they aren't user accessible

Reply:
Thanks for the clarification, Russ. I was kind of confused when the comment said this was only for mosaic-ing and the laplace crrej was in the previous step. I acknowledge this step is probably more complex so I'll omit it as you suggested.

The `WR96_DRIZZLE_PARAMS` are the same params that were there before, I just exposed them so that I could test the mosaic-ing in `test_drizzle.py`. Once we implement the CR rejection you detail here, I can add another test for the CR rejection as well